pkg/actions/addon/addon.go

```diff
@@ -37,9 +44,45 @@ func New(clusterConfig *api.ClusterConfig, eksAPI eksiface.EKSAPI, stackManager
 		oidcManager:   oidcManager,
 		stackManager:  stackManager,
 		clientSet:     clientSet,
+		timeout:       5 * time.Minute,
```

Comment:
all of the addons so far are extremely quick to deploy, so going with this for the moment

Reply:
Thanks for the wait flag, it would be greate to wait for a specified time. created an [issue](https://github.com/weaveworks/eksctl/issues/3642).